packages/design-system-scripts/gulp/docs/generatePages/processKssSection.js

```diff
@@ -92,6 +92,11 @@ function processFlags(section) {
             // KSS converts the URL to an <a> element, so we grab just the URL
             section.uswds = hrefUrl(value);
             break;
+          // case 'cmsds':
```

Comment:
Can we remove these?

Reply:
Yeah - I missed those the first time, thanks!